app/src/main/java/org/bepass/oblivion/OblivionVpnService.java

```diff
@@ -430,26 +430,50 @@ public void setLastKnownState(ConnectionState lastKnownState) {
         publishConnectionState(lastKnownState);
     }
 
+    private String getNotificationText() {
+        if(fileManager.getBoolean("USERSETTING_psiphon")) {
+            String countryCode = fileManager.getString("USERSETTING_country");
+            String countryName;
+            if("".equals(countryCode)) {
+                countryName = "Automatic";
+            } else {
+                countryName = CountryUtils.getCountryName(countryCode);
+            }
```

Comment:
Something like this looks a little cleaner:
```java
String countryName = "Automatic";
if (!countryCode.equals("")) {
  countryName = CountryUtils.getCountryName(countryCode);
}
```

Reply:
Done!